data/oyepriyansh.json

```
@@ -0,0 +1,238 @@
+{
+
+	"name": "Priyansh Prajapat",
+
+	"type": "personal",
+
+	"bio": "Aspirant Full stack Web Developer",
+
+
+
+	"socials": [
+
+		{
+			"icon": "FaInstagram",
+			"url": "https://instagram.com/oyepriyansh"
+		},
+
+		{
+			"icon": "FaTwitter",
+			"url": "https://twitter.com/oyepriyansh"
+		},
+
+		{
+			"icon": "FaGithub",
+			"url": "https://github.com/oyepriyansh"
+		},
+
+	],
+
+
+
+	"links": [
+
+		{
+
+			"group": "Overview",
+
+			"name": "Portfolio",
+
+			"url": "https://oyepriyansh.pages.dev",
+
+			"icon": "FaGlobe"
+
+		},
+
+		{
+
+			"group": "Overview",
+
+			"name": "Minifolio",
+
+			"url": "https://oyepriyansh.github.io",
+
+			"icon": "FaLink"
+
+		},
+
+		{
+
+			"group": "Overview",
+
+			"name": "Email",
+
+			"url": "mail to:hi@priyansh.app",
+
+			"icon": "SiGmail"
+
+		},
+
+		{
+
+			"group": "Socials",
+
+			"name": "Instagram:Follow me",
+
+			"url": "https://instagram.com/oyepriyansh",
+
+			"icon": "FaInstagram",
+
+			"color": "#cc0066"
+
+		},
+
+		{
+
+			"group": "Socials",
+
+			"name": "Twitter: Follow me",
+
+			"url": "https://twitter.com/oyepriyansh",
+
+			"icon": "FaTwitter",
+
+			"color": "#00ACEE"
+
+		},
+
+		{
+
+			"group": "Socials",
+
+			"name": "YouTube: Subscribe :)",
+
+			"url": "https://youtube.com/channel/UCmLi-IDLR5l7ZAlrSTd4LqQ",
+
+			"icon": "FaYoutube",
+
+			"color": "#FF0000"
+
+		},
+
+		{
+
+			"group": "Socials",
+
+			"name": "GitHub:Collaborateron Open Source",
+
+			"url": "https://github.com/oyepriyansh",
+
+			"icon": "FaGithub"
+
+		},
+
+		{
+
+			"group": "Socials",
+
+			"name": "LinkedIn: Let's connect",
+
+			"url": "https://linkedin.com/in/oyepriyansh",
+
+			"icon": "FaLinkedin",
+
+			"color": "#0033cc"
+
+		},
+
+		{
+
+			"group": "Socials",
+
+			"name": "Discord: Let`s hang out",
+
+			"url": "https://discord.com/invite/AeAjegXn6D",
+
+			"icon": "FaDiscord"
```

There is a missing comma `,`

```suggestion
			"icon": "FaDiscord",
```